### ISSUE TITLE
Error creating template

### ISSUE THREAD

**Issue by MrGordonElliott** [crimcol/Alt.FluentMigrator.VStudio#7]
Hi.  

I get the following error on 
Microsoft SQL Server 2017 (RTM) - 14.0.1000.169 (X64)   Aug 22 2017 17:04:49   Copyright (C) 2017 Microsoft Corporation  Express Edition (64-bit) on Windows 10 Home 10.0 <X64> (Build 18362: ) 

Add-FluentMigration InitialMigration -AddScript
'MyProject.Migrations' project build success.
Add-FluentMigration : Exception setting "Value": "Object of type 'System.Int32' cannot be converted to type 'System.UInt32'."
At line:1 char:1
+ Add-FluentMigration InitialMigration -AddScript
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Add-FluentMigration], SetValueInvocationException
    + FullyQualifiedErrorId : ExceptionWhenSetting,Add-FluentMigration

It creates an empty script file but no migration file and include for the script.

Am i missing something

**Comment by crimcol** [comment]
Hi @MrGordonElliott,
Could you please show your **migrations.json** file?
And what version of Visual Studio do you have?

**Comment by MrGordonElliott** [comment]
I am using Microsoft Visual Studio Community 2019

My migrations.json is:
{
  "ConnectionProjectName": "MyProject.Migrations",
  "ConnectionName": "Server=.;Database=Pulse2;Trusted_Connection=true;",
  "FluentMigrationToolPath": ".\\packages\\FluentMigrator.Console.3.2.6\\net461\\x86\\Migrate.exe",
  "DbProvider": "SqlServer",
  "DbProviderHelpUrl": "https://fluentmigrator.github.io/articles/runners/runner-console.html#--provider---dbtype---dbvalue-required",
  "MigrationFolder": "Migrations",
  "ScriptsFolder": "Scripts",
  "TimeFormat": "yyyyMMddHHmmss"
}

I have noticed that I cannot find 'Migrate.exe' even though i have installed  FluentMigrator.Console in the project

**Comment by crimcol** [comment]
@MrGordonElliott,
I can see only one wrong value in your migration.json file:
```
"ConnectionName": "Server=.;Database=Pulse2;Trusted_Connection=true;",
```
You have connection string instead of connection name.
Connection String should be in your project "MyProject.Migrations" (hope this is a name of project but not solution + project name) in App.config or Web.config file.

For example, in my solution I have two projects: ConsoleApp and DbMigrations.
ConsoleApp project has App.config and it contains ConnectionString with name TestDb to be able to connet to database using EntityFramework. Note: App.config file can have multiple connection strings.

DbMigrations project has migrations.json file with configuration:
ConnectionProjectName = ConsoleApp
ConnectionName = TestDb
Using configuration above we can know where connection string is located.

This is my App.config in ConsoleApp project:
```
<?xml version="1.0" encoding="utf-8" ?>
<configuration>
  <connectionStrings>
    <add name="TestDb" connectionString="Data Source=.\SQLEXPRESS2014;Initial Catalog=TestDb;Integrated Security=SSPI;" providerName="System.Data.SqlClient" />
  </connectionStrings>
  <startup>
    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8" />
  </startup>
</configuration>
```

**Comment by crimcol** [comment]
> I have noticed that I cannot find 'Migrate.exe' even though i have installed FluentMigrator.Console in the project.

I notice I need to update documentation. Please install FluentMigrator.Console package if you didn't installed. The latest version is 3.2.6.

Then you need to update migrations.json:
```
"FluentMigrationToolPath": ".\\packages\\FluentMigrator.Console.3.2.6\\tools\\net461\\x86\\Migrate.exe"
```

**Comment by MrGordonElliott** [comment]
Sorry for the delay.. busy at work.

So, can i use it as a stand alone project then ?  I thought if i supplied a connection string then fluentMigrator would be able to connect and  migrations would run.

My initial reported issue is that when your project successfully creates a timestampe_script.sql in the scripts folder, but then does not create the corresponding timestamp_migration.cs in the migrations folder with the code to run the script as below

            Execute.Script(@"Scripts\20200422213431_InitialMigration.sql");

Thanks for all your help.  I think if i could get this working then it would be of great help



________________________________

**Comment by MrGordonElliott** [comment]
My migration app with fluentMigrator nuget installed is is .NET core 3.1

________________________________

**Comment by crimcol** [comment]
@MrGordonElliott, it should work in .net core without any issues.
Please let me know if suggestions above works for you.

**Comment by crimcol** [comment]
Package updated according to the current latest FluentMigrator.Console 3.2.7 version.

**Comment by polzka90** [comment]
i am running in the same problem when i try to run using the -AddScript parameter, it will create the first .sql file but it given an error when try to create the second .sql

i am using the same migration.json of the readme, i am not pretending to run a Update-FluentDatabase, i just want to create a migration with the scripts via cmd for now

follow the error

Add-FluentMigration : Exception setting "Value": "Object of type 'System.Int32' cannot be converted to type 'System.UInt32'."
At line:1 char:1
    + Add-FluentMigration InitialMigration -AddScript
    + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Add-FluentMigration], SetValueInvocationException
    + FullyQualifiedErrorId : ExceptionWhenSetting,Add-FluentMigration

**Comment by crimcol** [comment]
> i am running in the same problem when i try to run using the -AddScript parameter, it will create the first .sql file but it given an error when try to create the second .sql
> 
> i am using the same migration.json of the readme, i am not pretending to run a Update-FluentDatabase, i just want to create a migration with the scripts via cmd for now
> 
> follow the error
> 
> Add-FluentMigration : Exception setting "Value": "Object of type 'System.Int32' cannot be converted to type 'System.UInt32'."
> At line:1 char:1
> + Add-FluentMigration InitialMigration -AddScript
> + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> + CategoryInfo : NotSpecified: (:) [Add-FluentMigration], SetValueInvocationException
> + FullyQualifiedErrorId : ExceptionWhenSetting,Add-FluentMigration

Hello @polzka90,
Could you please share your migrations.json file?
Ideally it would be nice if you can make simple project and attach, so I will be able to reproduce the issue.

**Comment by polzka90** [comment]
HI @crimcol i cant attach the project, but here is the migrations.json
{
  "ConnectionProjectName": "test",
  "ConnectionName": "TestDb",
  "FluentMigrationToolPath": ".\\packages\\FluentMigrator.Console.3.2.15\\tools\\net461\\x86\\Migrate.exe",
  "DbProvider": "SqlServer",
  "DbProviderHelpUrl": "https://fluentmigrator.github.io/articles/runners/runner-console.html#--provider---dbtype---dbvalue-required",
  "MigrationFolder": "Migrations",
  "ScriptsFolder": "Scripts",
  "TimeFormat": "yyyyMMddHHmmss"
}

i am using FluentMigrator.Console.3.2.15

**Comment by crimcol** [comment]
@polzka90, I will create a test working project with FluentMigrator.Console.3.2.15, so you will be able to compare with your solution.

**Comment by crimcol** [comment]
@polzka90,
Please check "[Test Project](https://github.com/crimcol/Alt.FluentMigrator.VStudio/tree/SampleProject/sample/ProjectExample)" branch in the **SampleProject** branch.
You can download the solution and test your commands.

**Comment by polzka90** [comment]
hi @crimcol i download your solution and test it in my personal computer, i think the problem is that i am using a Net Core project, i attach it here 
[Testing.zip](https://github.com/crimcol/Alt.FluentMigrator.VStudio/files/6174366/Testing.zip)

**Comment by crimcol** [comment]
@polzka90, I see the issue. It's not supported for .NET Core.
But I'm going to make an update, so it will work.

**Comment by crimcol** [comment]
Issue solved in **v1.3.2**.
